Makefile

```diff
@@ -19,8 +19,8 @@ MAC       := "Darwin"
 PACKAGES  := $$(go list ./...| grep -vE 'vendor')
 FILES     := $$(find . -name '*.go' | grep -vE 'vendor')
 
-LDFLAGS += -X "github.com/pingcap/tidb/util/printer.TiDBBuildTS=$(shell date -u '+%Y-%m-%d %I:%M:%S')"
-LDFLAGS += -X "github.com/pingcap/tidb/util/printer.TiDBGitHash=$(shell git rev-parse HEAD)"
+LDFLAGS += -X "github.com/alston111111/tidb/util/printer.TiDBBuildTS=$(shell date -u '+%Y-%m-%d %I:%M:%S')"
```

Comment:
Oh. Sorry about that.  Makefile has been changed back.

I was supposed to change it back before the push, but I forgot to.
I had already put my work in ${some_path}/github.com/pingcap.
And I will not do this again and I will pay attention to it.

Thank you.